### PR TITLE
Get-DbaPageFileSetting : Redesign

### PR DESCRIPTION
**Pull request by FriedrichWeinmann**
## Type of Change

 - [ ] Bug fix (non-breaking change, fixes #<enter issue number>)
 - [x] New feature (non-breaking change, adds functionality)
 - [ ] Breaking change (effects multiple commands or functionality)
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] Nunit test is included
 - [ ] Documentation
 - [ ] Build system
 
### Changes

 - Removed arbitrary host resolution
 - Removed hard OS 6.0+ barrier
 - Implemented dbatools coding standards
 - Switched to `Get-DbaCmObject`

### CODE REVIEW

File: functions/Get-DbaPageFileSetting.ps1 [hunk 1]
```diff
@@ -1,159 +1,108 @@
...
+        Get-DbaPageFileSetting -ComputerName ServerA,ServerB
+        
+        Returns a custom object displaying ComputerName, AutoPageFile, FileName, Status, LastModified, LastAccessed, AllocatedBaseSize, InitialSize, MaximumSize, PeakUsage, CurrentUsage  for ServerA and ServerB
+    
+    .EXAMPLE
+        'ServerA' | Get-DbaPageFileSetting
+        
+        Returns a custom object displaying ComputerName, AutoPageFile, FileName, Status, LastModified, LastAccessed, AllocatedBaseSize, InitialSize, MaximumSize, PeakUsage, CurrentUsage  for ServerA
+    
+    .NOTES
```
Comment (wsmelton) [inline functions/Get-DbaPageFileSetting.ps1]: Just reference the notes and links are put above the examples in almost all of the commands. Not sure if your template in PSS may reformat it this way?

File: functions/Get-DbaPageFileSetting.ps1 [hunk 2]
```diff
@@ -1,159 +1,108 @@
...
-    PROCESS {
-        foreach ( $Computer in $ComputerName ) {
-            $reply = Resolve-DbaNetworkName -ComputerName $Computer -Credential $Credential -ErrorAction silentlycontinue
-
-            if ( !$reply.FullComputerName ) {
-                # we can reach $computer
-                Write-Message -Level Warning -Message "$Computer is not available."
-                continue
+    process {
+        foreach ($Computer in $ComputerName) {
```
Comment (wsmelton) [inline functions/Get-DbaPageFileSetting.ps1]: `$Computer` would be lower case...some spots you already have it lowercase.

**Comment by FriedrichWeinmann** [comment]
Aye, the CBH formatting comes from my PowerShell Studio CBH templates, those have a different order so they were switched. Manually reordered things again, though that's not functional change.

Also updated the casing - I'm using a different casing convention in my own projects, so I forgot.
(I generally keep all variables dedicated to parameter input uppercase, as that makes it easier to keep track in ... lengthier functions. Obviously a non-issue for this function)

**Comment by potatoqualitee** [comment]
🌮 🌮 🌮 🌮 🌮 🌮 🌮

**Pull request merged**